.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,71 @@
+name: Token Dashboard
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+    - solidity/dashboard/**
+
+# For pull_request_target:
+#  - Checkout with repository set to PR repo, ref same (see
+#    https://github.com/actions/checkout#usage).
+#  - Trigger from issue comment?
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.11.x]
+
+    if: startsWith(github.head_ref, 'releases/mainnet/token-dashboard/')
```

Comment:
Do we want this workflow to eventually replace dashboard building in CircleCI for test environments? I understand that this branch prefix check will be moved to mainnet related steps.

Reply:
This will probably be the most straightforward way to deploy it across the board, yep---but I don't need us to get there immediately. I wanted to leave enough context for that work though.

For reference, this is almost identical to how we deploy the Keep and tBTC sites (see https://github.com/keep-network/website/blob/master/.github/workflows/build.yaml). I wanted the initial scope very limited until we can understand implications, though.